__tests__/lint/assertions-casing-camel-case-error/snapshot.js

```diff
@@ -3,7 +3,7 @@
 exports[`E2E lint assertions-casing-camel-case-error 1`] = `
 
 validating /openapi.yaml...
-[1] openapi.yaml:19:20 at #/paths/~1pet~1findByStatus/get/operationId
+[1] openapi.yaml:19:20 at #/paths/~1pet~1findByStatus/get/operationId/GetExampleId
```

Comment:
Something wrong her... This pointer seems invalid.

Reply:
There are many places like this and all of them seem to have extra segment at the end.

Reply:
I see. Not sure where does it come from though. Will take a look.

Reply:
Updated the snapshots. Please take a look.